app/src/main/java/com/tananaev/passportreader/MainActivity.kt

```diff
@@ -121,7 +121,7 @@ abstract class MainActivity : AppCompatActivity() {
             override fun onTextChanged(s: CharSequence, start: Int, before: Int, count: Int) {}
             override fun afterTextChanged(s: Editable) {
                 PreferenceManager.getDefaultSharedPreferences(this@MainActivity)
-                    .edit().putString(KEY_PASSPORT_NUMBER, s.toString()).apply()
+                    .edit().putString(KEY_PASSPORT_NUMBER, s.toString().uppercase()).apply()
```

Comment:
I think if we want to do this, we should probably do it in the UI level, so people can see what is being entered.

Reply:
Ok, I've pushed a new change now when you get a chance.